docker-compose.yml

```diff
@@ -4,7 +4,7 @@
 app:
   image: mageinferno/magento2-nginx:1.9.14-0
   links:
-    - phpfpm
+    - php:phpfpm
```

Comment:
I could, but why would you like to change it? It requires more changes in configuration file(s) (see `mageinferno/magento2-nginx` image). For example I could add `ENV PHP_HOST` or make ugly change default host in [mageinferno/docker-magento2-nginx/Dockerfile#L4](https://github.com/mageinferno/docker-magento2-nginx/blob/master/Dockerfile#L4).